apps/publish/content/correct.py

```diff
@@ -114,7 +114,7 @@ def on_update(self, updates, original):
         update_word_count(updates, original)
         flush_renditions(updates, original)
         self.change_being_corrected_to_published(updates, original)
-        self.send_to_original_desk(updates, original)
+        self.send_to_current_desk(updates, original)
```

Comment:
do we actually have to send it anywhere? it should be already there

Reply:
Good catch.